### PR TITLE
README Improvements

### PR DESCRIPTION
**Pull request by Rustmilian**
Various README Improvements :
Overall restructuring.
Added Hex info and image.
Added path info.
Added Playlist source images & gif.
Moved Banner image into nonvolatile project storage `assets`.
Added [!TIP] styling & key combo styling.
Added [!IMPORTANT] Firefox info and image.
Added image hover text.

### CODE REVIEW

**Comment by ccgauche** [comment]
That's very nice of you to improve the readme file if you need info or anything just ask for it

**Comment by Rustmilian** [comment]
@ccgauche matter of fact I wanted to ask if it was cool if I added my AUR package to the install section as explicitly a 3rd party package.

**Comment by Rustmilian** [comment]
@ccgauche https://github.com/Rustmilian/ytermusic/tree/master#YTerMusic

File: README.md [hunk 1]
```diff
@@ -1,44 +1,60 @@
...
+	- [ ] Custom theming
+
+## Install
+- Download the latest version from [releases](https://github.com/ccgauche/ytermusic/releases/latest)
+	### Linux
+	> 3rd party AUR packages are available [here](https://aur.archlinux.org/packages?O=0&K=ytermusic).
+	
+	Install the following libraries:
+	
+	```
```
Comment (williamdes) [inline README.md:40]:
```suggestion
	```sh
```

Comment (Rustmilian) [reply README.md:40]: That really doesn't make any difference. If it at least added syntax highlighting I'd might have done that, but whether or not its there is inconsequential.

Comment (Rustmilian) [reply README.md:40]: @ccgauche don't pull is quite yet. there's a small change I want to do.

**Comment by Rustmilian** [comment]
@ccgauche It's in a state I'm happy with now. If there's any changes you want feel free to say so.

**Comment by ccgauche** [comment]
> @ccgauche matter of fact I wanted to ask if it was cool if I added my AUR package to the install section as explicitly a 3rd party package.
> Also are there any more colors supported other then Blue, White, Green, Black, Red, & Yellow?

I'm ok with 3rd party distributions if they are cited as 3rd parties.
Yes we can technically use all RGB colors. I'm working on theming the app to make it more customizable.

Apart from that I'm pretty happy with the changes.

**Comment by Rustmilian** [comment]
> Yes we can technically use all RGB colors

Good to know. I figured how too, I'll make some changes. \
Also, I just want so that the download speed is way faster now. ❣️

**Comment by Rustmilian** [comment]
@ccgauche I think we're near the final product, give it one last look through and tell me whatcha think! 🥰

one last change needed is adding [paths](https://github.com/Rustmilian/ytermusic/tree/master#paths) for windows, then it can be merged.

**Comment by ccgauche** [comment]
If you wanna add paths you can or you can link to the directories doc with some info about which file is in which directories folder

**Comment by Rustmilian** [comment]
> If you wanna add paths you can or you can link to the directories doc with some info about which file is in which directories folder

This one? https://docs.rs/directories/latest/directories/struct.ProjectDirs.html#method.config_dir

**Comment by Rustmilian** [comment]
@ccgauche If that was the right place to link, it's ready for merge now.

**Review by ccgauche — APPROVED** [review]
Seems pretty good to me I'm very happy with the result. Thanks a lot for your help and I hope to see you contribute to the project again!

**Review by williamdes — COMMENTED** [review]
Review request to cleanup https://github.com/pulls

**Pull request merged**